manifests/m/MaëlHörz/HxDHexEditor/2.5/MaëlHörz.HxDHexEditor.installer.yaml

```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: MaëlHörz.HxDHexEditor
+PackageVersion: 2.5
+Installers:
+- Architecture: neutral
+  InstallerType: inno
+  InstallerUrl: https://headblockhead.com/files/winget/HxDSetup.exe

```

Comment:
I can't find any mention on the [HxD website](https://mh-nexus.de/) that this is an official/trustworthy mirror. I am not sure what the policy is regarding unofficial mirrors. Maybe we should wait until WinGet supports installers in ZIP archives which would allow using the official downloads.

Reply:
Ok, I'll close this pull request. Hopefully zip support comes soon!